image/image_test.go

```diff
@@ -884,12 +848,13 @@ func (s *imageSuite) TestSetupSeedWithBase(c *C) {
 	model := s.Brands.Model("my-brand", "my-model", map[string]interface{}{
 		"architecture":   "amd64",
 		"gadget":         "pc18",
-		"kernel":         "pc-kernel",
+		"kernel":         "pc-kernel=18",
```

Comment:
I should probably undo this change, this is actually tested in TestSetupSeedWithKernelAndGadgetTrack